_configtest.o.d

```diff
@@ -0,0 +1,4 @@
+commandline: -I/home/tmabbot/anaconda3/include/python3.5m -c-I/home/tmabbot/anaconda3/include/python3.5m
```

Comment:
I don't know what sort of development setup you use, but I'd guess this is not the usual way to do it, so I can't offer advice. In any case, you shouldn't include these files in the PR.